### DISCUSSION TITLE
Schedule email with payment in report

### DISCUSSION THREAD

**Discussion by micooro** [YetiForceCompany/YetiForceCRM#15459] [How to] [answered]
Hello to all.
I need to send an email with all "payment in" that expired in next 10 days.
In workflow i tried but i receive an email for each payment.
I would like a single email with in a body all "payment in" that expired.
Any idea about?
Thanks a lot.

Michele

**Comment by vovpff** [comment]
I has the same task for Leads. This is not ready (and not started) for this time.

I think there is no function available from the box for this task.

I want to implement it in futere as custom workflow function with custom textparser function inside email template

**Comment by rskrzypczak** [comment]
You can get it with the help of the textparser.
![image](https://user-images.githubusercontent.com/10221999/142914858-3499897f-7321-4414-b8a7-043279c48dc1.png)
![image](https://user-images.githubusercontent.com/10221999/142914690-5077a9ad-c8b7-4e06-afe5-963a6f0fae4b.png)
And all you have to do is add the appropriate textparser function to the e-mail template

**Comment by micooro** [comment, marked as answer]
I found the solution with textparser

$(recordsList : PaymentsIn|||PRIC15D)$

Where "PRIC15D" is my the view in Payments IN.

Then i scheduled a workflow with enail report's task
Thanks to all for the suggestions
